### PR TITLE
Task with due date time 3

### PR DESCRIPTION
**Pull request by SafeerAhmFarooqi**

### CODE REVIEW

File: resources/views/task.blade.php [hunk 1]
```diff
@@ -3,35 +3,90 @@
...
                 <h5 class="card-header">New Task</h5>
                 <div class="card-body">
-                    <form action="{{route('task.store')}}" method="POST">
+                    <form action="{{route('task.store')}}" method="POST" class="row g-3">
                         @csrf
                         <div class="mb-3">
-                          <label for="exampleInputEmail1" class="form-label">Task</label>
-                          <input type="text" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" name="name">
+                          <label for="task" class="form-label">Task</label>
+                          <input type="text" class="form-control {{$errors->has('name')? 'is-invalid' : (old('name')? 'is-valid' : '')}}" id="task" aria-describedby="emailHelp" name="name" value="{{old('name')}}">
```
Comment (amanank) [inline resources/views/task.blade.php:18]: what is `aria-describedby="emailHelp"`?

**Review by amanank — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -21,26 +22,32 @@ public function store(Request $request)
         $request->validate([
             //Validation Rules  
             'name' => ['required','max:255'],
+            'due_date' => ['required','max:255'],
+            'due_time' => ['required','max:255',new TaskUniqueDateTime($request->due_date)],
```
[the remark this answers is not in the document]
Comment (SafeerAhmFarooqi) [reply app/Http/Controllers/TaskController.php:26]: i have changed it to 5

File: app/Http/Controllers/TaskController.php [hunk 3]
```diff
@@ -21,26 +22,32 @@ public function store(Request $request)
         $request->validate([
             //Validation Rules  
             'name' => ['required','max:255'],
+            'due_date' => ['required','max:255'],
```
[the remark this answers is not in the document]
Comment (SafeerAhmFarooqi) [reply app/Http/Controllers/TaskController.php:25]: I have changed it to 10

File: resources/views/task.blade.php (hunk 1 above)
Comment (SafeerAhmFarooqi) [reply resources/views/task.blade.php:18]: I have removed it.

**Pull request merged**